README.md

```diff
@@ -273,14 +292,12 @@ block's partial config.
 </details>
 
 <details>
-<summary>How to compose blocks? (a.k.a. building presets)</summary>
+<summary>How to compose blocks? (AKA building presets)</summary>
```

Comment:
Ok, let's just remove it in this particular case.

(I liked pointing out the "presets" use case, but as I think about it, "composing" seems better)